TRANSIENT-SETTINGS.md

```diff
@@ -168,6 +168,7 @@ All execution-related settings are available for transient units.
 ✓ ReadWritePaths=
 ✓ ReadOnlyPaths=
 ✓ InaccessiblePaths=
+✓ EmptyDirectories=
```

Comment:
the commit's title is wrong btw, it says "ExecDirectories=", but should say `EmptyDirectories=`